rust/operator-binary/src/crd.rs

```diff
@@ -65,11 +65,18 @@ pub struct AutoTlsCa {
 pub struct KerberosKeytabBackend {
     pub realm_name: Hostname,
     pub kdc: Hostname,
-    pub admin_server: Hostname,
+    pub admin: KerberosKeytabBackendAdmin,
     pub admin_keytab_secret: SecretReference,
     pub admin_principal: KerberosPrincipal,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase")]
+pub enum KerberosKeytabBackendAdmin {
+    #[serde(rename_all = "camelCase")]
+    Mit { admin_server: Hostname },
```

Comment:
What do you think of calling it simply `server`, as the field above is already called `admin`?

Reply:
I was considering calling it `kadmin_server`, since I want to call out the protocol specifically (as opposed to AD's `ldap_server`).

Reply:
`kadmin_server` would be great!

Reply:
0b6fd37